src/backend/cuda/platform.hpp

```diff
@@ -74,14 +72,14 @@ bool& evalFlag();
 
 class MemoryManager;
 class MemoryManagerPinned;
-class InteropManager;
 ///////////////////////// BEGIN Sub-Managers ///////////////////
 //
 MemoryManager& getMemoryManager();
 
 MemoryManagerPinned& getMemoryManagerPinned();
 
-InteropManager& getGfxInteropManager();
+typedef common::InteropManager<GraphicsResourceManager, CGR_t> GraphicsManager;
```

Comment:
I don't think this is necessary. Can you try renaming `GraphicsResourceManager` to `GraphicsManager` and removing this line to see if it works?